.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Slimefun
+This document outlines various ways how you can help contribute to Slimefun and make this a bigger and better project.<br>
+All contributions must be inline with our [Code of Conduct](https://github.com/TheBusyBiscuit/Slimefun4/blob/master/.github/CODE_OF_CONDUCT.md) and [License](https://github.com/TheBusyBiscuit/Slimefun4/blob/master/LICENSE).
+Please also follow the templates for Issues and Pull Requests we provide.
+
+## 1. Issues: Bug Reports
+One of the foundations for good software is reliability. To facilitate this reliability, our community must work together to crush bugs that arise. 
+This of course requires good information and knowledge about ongoing bugs and issues though.
```

Comment:
I think this line could use some punctuation too like "This, of course, requires..."

Reply:
A paranthesis would be wrong in this use case here.